### ISSUE TITLE
Manual operating z-wave devices results in status of devices are not updated in Homey

### ISSUE THREAD

**Issue by aljoen** [athombv/homey#1235]
**What is your Homey version (Settings → System)?**

1.1.2

**What did you try to achieve?**

Manual operating z-wave switches, e.g.  on/off, results in out of sync status in Homey.

*Please include detailed steps how we can reproduce the issue

Using buttons on the plugs,dimmers itself to switch on/off do work. However the status in homey is not updated. This happens for all my z-wave devices such as Fibaro plugs, dimmers and also for other brand of switches 

**What was the result?**
If manual operating switches, dimmers etc,  homey reports no status change, e.g.lamp is manual turned on, homey still reports lamp is off. 
Result is that flows depending on states of switches,dimmers etc do not work correctly
 
...

**What did you expect as result?**
Z-wave is 2 way communication. Manual operating devices should be in-sync within homey
...

**Any other remarks**

...

**Comment by Inversion-NL** [comment]
Works for me. What devices so you use?

**Comment by aljoen** [comment]
This happens  basically for all my z-wave devices, such as:
Fibaro Dimmer FGD-211
Fibaro Wall Plug FGWPE-101
Popp Wall Plug 123580 67G
Everspring Wall Plug AD1422

**Comment by RobinBol** [comment]
That is strange, could you please to to place Homey closer to some of your nodes and test whether the `onoff` status changes when you toggle the Fibaro Wall Plug for example?

**Comment by aljoen** [comment]
That did not work either
However I installed some new Neo plugs and they reported correct. Did a factory reset of Fibaro plugs (could not exclude them..what ever I did, but thats an other issue) and re-added. Again it worked. Same with dimmer 2, re-adding did work. However excluden en re-adding a Popp dimmer plug did not work, strange
Seems I have to exclude and re add all of my z-wave device... :( At least it is working now

Only thing I can think off, had a veralite and z-wave remote as secondary controller. Maybe that did make "reporting class"go haywire. BTW all associations of device are properly set to 1 (Homey) 

For me issue is closed
